apps/wallet-mobile/src/features/Portfolio/useCases/PortfolioTokenDetails/PortfolioTokenDetailsScreen.tsx

```diff
@@ -72,37 +71,38 @@ export const PortfolioTokenDetailsScreen = () => {
   }, [activeTab, setActiveTab, strings.overview, strings.performance, strings.transactions, styles.tab, styles.tabs])
 
```

Comment:
if (activeTab !== 'transactions') return null

Reply:
This is the whole screen, not just the transaction's list.
In order for the scroll to work nicely, the whole screen is made with the list, adding a lot of it in the header part.
I don't want to return null to the screen when active tab is not transactions, just the data inside the list.
It's how the vendor solved the flickering of having 2 nested scrolls.